e2e/staticpvc.go

```diff
@@ -189,6 +189,102 @@ func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock, checkI
 	return err
 }
 
+func validateRBDStaticMigrationPV(f *framework.Framework, appPath string, isBlock, checkImgFeat bool) error {
```

Comment:
`checkImgFeat` not used in this function, needs to be removed.

Reply:
:+1: , done.. thanks